pkg/apis/componentconfig/types.go

```diff
@@ -274,6 +274,8 @@ type KubeletConfiguration struct {
 	NodeIP string `json:"nodeIP,omitempty"`
 	// nodeLabels to add when registering the node in the cluster.
 	NodeLabels map[string]string `json:"nodeLabels"`
+	// Traffic to IPs outside this range will use IP masquerade.
```

Comment:
OK - I sort of struggled to phrase it because of the negative definition, but I will try!